ONSdigital/rm-reporting
# RAS 958: Upgrade reporting to Flask 3

# What and why?
The pinned version of Flask needed updating to version 3. This PR will achieves this.

# How to test?
Build and test locally. Push changes to your env and check everything works as expected.

# Jira
https://jira.ons.gov.uk/browse/RAS-926
https://jira.ons.gov.uk/browse/RAS-958
